nwbwidgets/controllers/group_and_sort_controllers.py

```diff
@@ -254,7 +254,7 @@ def order_dd_observer(self, change):
         if change["name"] == "value":
             self.order_by = self.order_dd.value
 
-            order_vals = self.get_column_values(by=self.order_by)
+            order_vals = self.get_column_values(column_name=self.order_by)
```

Comment:
this was a small unrelated fix that got into this PR somehow.  I didn't bother making another PR just for this.